x-pack/platform/plugins/shared/onechat/server/services/chat/types.ts

```diff
@@ -61,4 +66,9 @@ export interface ChatConverseParams {
    * These tools will be registered as LLM tools with browser.* namespace.
    */
   browserApiTools?: BrowserApiToolMetadata[];
+  /**
+   * Optional runtime configuration overrides.
+   * These override the stored agent configuration for this execution only.
+   */
+  configurationOverrides?: AgentConfigurationOverrides;
```

Comment:
Do we want this for the chat/converse API, or only for direct agent execution?

Reply:
only direct execution for now i think

Reply:
will remove this